source/puppet/4.3/reference/release_notes_agent.md

```diff
@@ -62,6 +79,17 @@ We no longer provide `puppet-agent` packages that will install on Windows Server
 
 * Resolves [a daemonization issue on AIX](https://tickets.puppetlabs.com/browse/PA-67) that made the service appear to be inoperative when running.
 
+### Regressions
+
+#### Smaller CA Certificate Bundle Can Cause Failures
+
+Through version 1.2.7, the `puppet-agent` package used the system-provided OpenSSL certificate authority (CA) certificate bundle on platforms where it was available, such as on Linux-based operating systems. Starting with `puppet-agent` 1.3.0, the package includes a CA certificate bundle to support encrypted access consistently across platforms, including those that don't provide their own certificate bundle.
```

Comment:
minor nit, instead of "encrypted access" I'd say "to support authenticated SSL connections consistently across platforms"